### PR TITLE
Add versioning and labels to SMPS, and write the protobuf file

### PR DESCRIPTION
**Pull request by kanishkarj**
This PR adds versioning and test labels to the SMPS. @leecalcote does the versioning and labels go into the test results too, i.e. in `example/service mesh performance specification result.yaml` ? 

Signed-off-by: kanishkarj <kanishkarj@hotmail.com>

### CODE REVIEW

**Comment by kanishkarj** [comment]
I have added the protobuf file too. @leecalcote @nveenjain

**Comment by kanishkarj** [comment]
Just wondering, do we need the `profile` field now that we have `labels` ?

**Comment by kanishkarj** [comment]
@nveenjain Thanks for the review, I shall fix them. Also, please do check if I have used the correct types, especially in places where `repeated` had to be used.

File: smps.proto [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    // experiments have different standard mesh profiles
+    string profile = 9;
+    
+    // location of posted results
+    string details_uri = 10;
+    
+    // URL of the service that is getting tested (getting load thrown at it)
+    string endpoint_url = 11;
+    
+    repeated Label labels = 12;
```
[the remark this answers is not in the document]
Comment (kanishkarj) [reply smps.proto:165]: Yes! thanks for pointing it out. Have made the changes.

**Comment by nveenjain** [comment]
@kanishkarj , the DCO check is failing...

**Comment by kanishkarj** [comment]
Sorry, I fixed it.

File: smps.proto [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    // Only sidecar metrics are measured
+    message IndividualWorkload {
+        string name = 1;
+        int32 count = 2;
+        int32 cpu_mCores = 3;
+        int32 mem_mb = 4;
+    }
+    
+    // Spec version
+    string smps_version = 1;
```
Comment (leecalcote) [inline smps.proto:140]: Gents, what say you to starting the spec version at v0.1.0? Or something much less that v1.0?

Comment (kanishkarj) [reply smps.proto:140]: Hmm.. true.. We have a lot of things to finalize

Comment (kanishkarj) [reply smps.proto:140]: Also, something we want to clarify would be that which of these elements are lists?

Comment (nveenjain) [reply smps.proto:140]: I second that @leecalcote !!! versioning should start at "v0.1.0"

**Pull request merged**